Specs/DateTime/Chinese/DateTimeModel.json

```diff
@@ -3240,5 +3240,77 @@
         }
       }
     ]
+  },
+  {
+    "Input": "18年10月1日到19年2月3日",
```

Comment:
Can you add at least one equivalent Japanese test case too? Even if not implemented yet, it's good to have the spec to test the future implementation.